addons/repair/models/product.py

```diff
@@ -40,4 +40,5 @@ def _count_returned_sn_products_domain(self, sn_lot, or_domains):
 class ProductTemplate(models.Model):
     _inherit = "product.template"
 
-    create_repair = fields.Boolean('Create Repair', help="Create a linked Repair Order on Sale Order confirmation of this product.", groups='stock.group_stock_user')
+    service_tracking = fields.Selection(selection_add=[('repair', 'Repair')],
```

Comment:
```suggestion
    service_tracking = fields.Selection(selection_add=[('repair', 'Repair Order')],
```
As it's called `Repair Order` everywhere else.